webapp_settings/production.py

```diff
@@ -59,13 +56,20 @@
 EMAIL_SIGNUP_ENDPOINT = 'https://democracyclub.org.uk/mailing_list/api_signup/v1/'
 EMAIL_SIGNUP_API_KEY = '{{ vault_email_signup_api_key }}'
 
-import sentry_sdk
-from sentry_sdk.integrations.django import DjangoIntegration
+def get_env():
+    tags = InstanceTags()
+    server_env = None
+    if tags['Env']:
+        server_env = tags['Env']
+
+    if server_env not in ['test', 'prod', 'packer-ami-build']:
```

Comment:
I wasn't sure if we also need a `stage` env here? I'm not sure if when you run a staging deploy if an explicit `ENVIRONMENT=stage` or similar is used, or if you allow the default `test` to be used https://github.com/DemocracyClub/who_deploy/blob/master/aws.yml#L18